### PR TITLE
Fix Japanese messages.

### PR DESCRIPTION
**Pull request by orihalcon128**
Fix Japanese messages.

### CODE REVIEW

**Comment by KengoTODA** [comment]
Thanks @orihalcon128 for your contribution!
I found some changes which omits `<code>` element. Could you restore these elements, or is it expected change? For instance:

```patch
diff --git a/spotbugs/etc/messages_ja.xml b/spotbugs/etc/messages_ja.xml
index 5b39507..d0e7916 100644
--- a/spotbugs/etc/messages_ja.xml
+++ b/spotbugs/etc/messages_ja.xml
@@ -516,7 +516,7 @@ FindNullDeref ディテクタで、非 null 値だけが使われるべきコン
     <Details>
 <![CDATA[
 <p>
-このディテクタは、クラスが別のパッケージのクラスによって拡張されるなら、予想外の結果をもたらす可能性がある this.getClass().getResource(...) の使用を探します。
+このディテクタは、クラスが別のパッケージのクラスによって拡張されるなら、予想外の結果をもたらす可能性がある <code>this.getClass().getResource(...)</code> の使用を探します。
 </p>
 ]]>
     </Details>
```

**Comment by KengoTODA** [comment]
If possible please fix remaining strange Japanese such as:

 *  `メソッドの呼び出を探します` (better to use `呼び出し`)
 * `長さが0の配列を返すことは、このコンテキストにおいて null 参照を返すことが一般的に好ましいです` (better to say `null参照を返すこと *よりも* 一般的に好ましい`)

**Comment by KengoTODA** [comment]
`messages.xml` (in English) also has `<blockquote><pre>` tags. To keep consistency, I recommend you to fix it not only in `messages_ja.xml` but also `messages.xml` and `messages_fr.xml`.

**Comment by orihalcon128** [comment]
I correct the comments <code> tag and strange Japanese messages.

Fix the <blockquote> tag creates another request.

File: spotbugs/etc/messages_ja.xml [hunk 1]
```diff
@@ -4051,15 +4104,16 @@ public static メソッドは、クラスの 静的な状態の一部である
...
+<code>wait</code> メソッドを呼び出すと，待機しているオブジェクトのロックを解除するだけで，その他のロックは解除しません。
 これは必ずしもバグではありませんが厳密に調べる価値があります。
 </p>
 ]]>
     </Details>
   </BugPattern>
 
-  <BugPattern type="TLW_TWO_LOCK_NOTIFY" deprecated="true"> <!-- never generated -->
+  <BugPattern type="TLW_TWO_LOCK_NOTIFY" deprecated="true">
+  <!-- never generated -->
```
Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: better to revert this change

File: spotbugs/etc/messages_ja.xml [hunk 2]
```diff
@@ -4381,99 +4440,99 @@ private Long getNotificationSequenceNumber() {
...
   <BugPattern type="MS_SHOULD_BE_FINAL">
     <ShortDescription>final にすべきフィールド</ShortDescription>
-    <LongDescription>{1} は、final ではありませんが final にすべきです。</LongDescription>
+    <LongDescription>{1} は，final ではありませんが final にすべきです。</LongDescription>
     <Details>
 <![CDATA[
 <p>
-final でない public static フィールドは悪意のあるコードや偶然別のパッケージによって変更できます。
-フィールドは、脆弱性を回避するために final にします。
+<code>final</code> ではない <code>public static</code> フィールドは悪意のあるコードや偶然別のパッケージによって変更される可能性があります。
```
Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: I think `could be changed by accident from another package` should be not `偶然別のパッケージによって変更される可能性がある` but `別のパッケージによって思いがけず変更される可能性がある`, how do you feel?

ここでaccident は偶然とかたまたまというよりは、クラス実装者が「意図しない」変更を指してるような気がします。

File: spotbugs/etc/messages_ja.xml [hunk 3]
```diff
@@ -5492,48 +5552,46 @@ static フィールドにすることを検討してください。
...
-<p>
-- メソッドは、副作用を持つかもしれないクラスのロードをトリガすることを意図している
-</p>
-<p>
-- メソッドは、暗黙のわかりにくい例外をスローするように意図されている
-</p>
+<ul>
+  <li>メソッドは，副作用を持つかもしれないクラスのロードをトリガーすることを意図している</li>
+  <li>メソッドは，暗黙のわかりにくい例外をスローするように意図されている</li>
+</ul>
```
Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: It's nice to fix this HTML problem even in `messages.xml` and `messages_fr.xml`

File: spotbugs/etc/messages_ja.xml [hunk 4]
```diff
@@ -99,72 +99,135 @@
...
+<![CDATA[
+<p>
+紛らわしいコード，異常なコード，またはエラーを引き起こす方法で書かれたコードです。
+たとえば，ローカル変数への無効な代入， <code>switch</code> 文のフォールスルー，未確認のキャスト， <code>null</code> とわかっている値の冗長な <code>null</code> チェックなどです。
+</p>
+<p>
+ より多くの誤検出を受け入れました。
+</p>
+<p>
+FindBugs の以前のバージョンでは、このカテゴリは Style として知られていました。
```
Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: still using `、`

File: spotbugs/etc/messages_ja.xml [hunk 5]
```diff
@@ -99,72 +99,135 @@
...
   </BugCategory>
 
   <BugCategory category="PERFORMANCE">
     <Description>性能</Description>
     <Abbreviation>P</Abbreviation>
-    <Details>必ずしも間違っているというわけではなくて、効率が悪いかもしれないコード</Details>
+    <Details>
+<![CDATA[
+<p>
+必ずしも間違っているというわけではないが、効率が悪いかもしれないコードです。
```
Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: still using `、`

File: spotbugs/etc/messages_ja.xml [hunk 6]
```diff
@@ -99,72 +99,135 @@
   <BugCategory category="CORRECTNESS">
     <Description>正確性</Description>
     <Abbreviation>C</Abbreviation>
-    <Details>バグの可能性 - おそらく、開発者が意図していなかったコードになっている明らかなコーディング間違いです。
-      我々は低い誤検出率のために努力します。
+    <Details>
+<![CDATA[
+<p>
+バグの可能性 - おそらく，開発者が意図していなかったコードになっている明らかなコーディングミスです。
```
Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: Could you tell me your intent to replace `、` with `， `?

I thought that you chose report type style, but you still use `。`

File: spotbugs/etc/messages_ja.xml [hunk 7]
```diff
@@ -2829,44 +2893,44 @@ URL の <code>equals</code> と <code>hashCode</code> は、ドメイン名の
...
 </p>
 ]]>
     </Details>
   </BugPattern>
 
   <BugPattern type="DM_EXIT">
-    <ShortDescription>System.exit(...) を呼び出しているメソッド</ShortDescription>
-    <LongDescription>{1} は、System.exit(...) を呼び出しています。これは Java 仮想マシン全体をシャットダウンさせてしまいます。</LongDescription>
+    <ShortDescription>System.exit(..) を呼び出しているメソッド</ShortDescription>
+    <LongDescription>{1} は，System.exit(..) を呼び出しています。これは Java 仮想マシン全体をシャットダウンさせてしまいます。</LongDescription>
```
Comment (orihalcon128) [inline spotbugs/etc/messages_ja.xml]: "..." is the Java keyword meaning varargs.
I thought ".." is better than "...".
Do I think too much?

File: spotbugs/etc/messages_ja.xml (hunk 2 above)
Comment (orihalcon128) [inline spotbugs/etc/messages_ja.xml]: I also think so.
I correct it.

File: spotbugs/etc/messages_ja.xml (hunk 7 above)
Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: Maybe yes, I think you can keep using `...`, it will not be mixed with varargs like `String...`.

Comment (orihalcon128) [inline spotbugs/etc/messages_ja.xml]: OK.

File: spotbugs/etc/messages_ja.xml (hunk 3 above)
Comment (orihalcon128) [inline spotbugs/etc/messages_ja.xml]: Improve HTML formatting.
- Modify source code block(&lt;blockquote&gt; -> &lt;code&gt;).
- Change to the bulleted list(&lt;ul&gt;&lt;/ul&gt;).
- Change Datails of the BugCategory into the HTML format.

Is there anything else you need to improve?

Comment (KengoTODA) [inline spotbugs/etc/messages_ja.xml]: I do not have another HTML problem to fix, thank you!
